docs/src/main/asciidoc/spring-cloud-gateway.adoc

```diff
@@ -1302,6 +1331,7 @@ public RouteLocator customRouteLocator(RouteLocatorBuilder builder, ThrottleGate
                         10,
                         TimeUnit.SECONDS)))
                 .uri("http://httpbin.org:80")
+                .metadataOption("key", "value")
```

Comment:
`MetadataOption` needs to be renamed in docs.

Reply:
done